tests/integration/components/conversations/status-select-test.js

```diff
@@ -0,0 +1,52 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import { set } from '@ember/object';
+import hbs from 'htmlbars-inline-precompile';
+import PageObject from 'ember-cli-page-object';
+import component from 'code-corps-ember/tests/pages/components/conversations/status-select';
+
+let page = PageObject.create(component);
+
+function renderPage() {
+  page.render(hbs`
+    {{conversations/status-select
+      status=status
+    }}
+  `);
+}
+
+moduleForComponent('conversations/status-select', 'Integration | Component | conversations/status select', {
+  integration: true,
+  beforeEach() {
+    page.setContext(this);
+  },
+  afterEach() {
+    page.removeContext();
+  }
+});
+
+test('it opens, closes, and renders the correct states', async function(assert) {
+  assert.expect(8);
+
+  set(this, 'status', 'open');
+
+  renderPage();
```

Comment:
does `renderPage` need an away?  I'm guessing not since this passes, but have seen `await` with render in the new qunit api

Reply:
It might need it of we're setting a component property without wrapping it in a `run`. Someone ought to try that out.